pyramid/config/views.py

```diff
@@ -804,22 +823,31 @@ def wrapper(context, request):
                 stacklevel=4,
                 )
 
+        if accept is not None:
+            if is_nonstr_iter(accept):
+                raise ConfigurationError(
+                    'A list is not supported in the "accept" view predicate.',
+                )
+            if '*' in accept:
+                warnings.warn(
+                    ('Passing a media range to the "accept" argument of '
+                     'Configurator.add_view is deprecated as of Pyramid 1.10. '
+                     'Use explicit media types to avoid ambiguities in '
+                     'content negotiation that may impact your users.'),
+                    DeprecationWarning,
+                    stacklevel=4,
+                    )
+            # XXX when media ranges are gone, switch verify=True
```

Comment:
Triple X the new "TODO"?

Reply:
Ha, I just saw this comment after reviewing. Forget mine. I have a broader meaning for TODO.

Is there an issue in the tracker attached to a milestone for a future release that covers this?

Reply:
I added https://github.com/Pylons/pyramid/issues/3383